underfs/s3/pom.xml

```diff
@@ -37,6 +37,30 @@
       <version>0.8.1</version>
     </dependency>
 
+    <!-- The following are jets3t dependencies
+     (see http://mvnrepository.com/artifact/net.java.dev.jets3t/jets3t/0.8.1).
+     We need to explicitly include them, because jets3t is shaded. -->
```

Comment:
Sorry I missed your comment before merging this in, I will open a new PR for it.